### PR TITLE
Modify rule S6656: Add missing message and highlighting info

### PR DESCRIPTION
**Pull request by egon-okerman-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by loris-s-sonarsource — APPROVED** [review]
LGTM !

File: rules/S6656/azureresourcemanager/rule.adoc [hunk 1]
```diff
@@ -18,4 +18,24 @@ include::how-to-fix-it/arm.adoc[]
...
+Change this code to not use an outer expression evaluation scope in nested templates.
+
+=== Highlighting
+==== Primary Highlight
+If `properties.expressionEvaluationOptions.scope` is set to an incorrect value, then highlight the `properties.expressionEvaluationOptions.scope` property.
+
+If `properties.expressionEvaluationOptions.scope` or `properties.expressionEvaluationOptions` is not set at all, then highlight `properties` entirely (of the `"Microsoft.Resources/deployments"` resource where it should be set.)
+
+==== Secondary Highlight
+Highlight the secure parameter in the nested template that is at risk here.
```
Comment (petertrr) [inline rules/S6656/azureresourcemanager/rule.adoc:39]: What message should be used here?

Comment (egon-okerman-sonarsource) [reply rules/S6656/azureresourcemanager/rule.adoc:43]: The PR is updated to contain the secondary message as well.

Comment (loris-s-sonarsource) [reply rules/S6656/azureresourcemanager/rule.adoc:43]: Still LGTM

**Review by jamie-anderson-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**